pro-planner-server/helpers/user.js

```diff
@@ -4,39 +4,59 @@
 const { ObjectId } = require('mongodb');
 const plan = require('../models/plan');
 
-async function addUser(eventId, userName) {
-  let users = await plan.findOneAndUpdate(
-    { _id: new ObjectId(eventId) },
-    {
-      $set: {
-        [`userInfo.${userName}`]: { id: new ObjectId() },
-      },
-    },
-    {
-      new: true,
-      projection: {
-        _id: 0,
-        userInfo: 1,
-      },
-    }
-  );
-  users = users.toJSON().userInfo;
-  return users;
+async function addUser(planId, userName) {
+	let users = await plan.findOneAndUpdate(
+		{ _id: new ObjectId(planId) },
+		{
+			$set: {
+				[`userInfo.${userName}`]: { id: new ObjectId() },
+			},
+		},
+		{
+			new: true,
+			projection: {
+				_id: 0,
+				userInfo: 1,
+			},
+		}
+	);
+	// users = users.toJSON().userInfo;
```

Comment:
Is this supposed to be commented out? The PUT request expects a return value from this function.

Reply:
That's interesting, i'm not sure how adding users is working without it, but I will uncomment it just in case.